swift/StableDiffusion/pipeline/StableDiffusionPipeline+Resources.swift

```diff
@@ -74,11 +76,22 @@ public extension StableDiffusionPipeline {
             FileManager.default.fileExists(atPath: urls.safetyCheckerURL.path) {
             safetyChecker = SafetyChecker(modelAt: urls.safetyCheckerURL, configuration: config)
         }
+        
+        // Optional Image Encoder
+        let encoder: Encoder?
+        if
+            let encoderModel = try? MLModel(contentsOf: urls.encoderURL, configuration: config)
```

Comment:
It would be good to distinguish between no model (missing file) and incorrect model (other error thrown).

Reply:
I did add a new error at first, but now that we implement the `ManagedMLModel` we now just test to see if there is an encoder model file before attempting to load it.